### ISSUE TITLE
Support sync frequency configuration

### ISSUE THREAD

**Issue by bastienbeurier** [NangoHQ/nango#21]
For simplicity sake, we are starting with a default sync frequency of 1h. It should be easy to (configure this sync frequency via: 
- The `sync` call (using the config object?)
- Later on with a CLI and dashboard

**Comment by rguldener** [comment]
Just Sync config would already help a lot :)

Also to consider:
We probably should not start a new refresh for a Sync before the last one has finished.

**Comment by bastienbeurier** [comment]
Right! I'll create another issue to discuss job timeout, retries, etc.
